docs/topics/email.txt

```diff
@@ -384,6 +385,16 @@ The class has the following methods:
   For MIME types starting with :mimetype:`text/`, binary data is handled as in
   ``attach()``.
 
+.. class:: EmailAttachment
+
+    A named tuple to store attachments to an email.
+
+    The named tuple has the following indexes:
+
+* ``filename``
+* ``content``
+* ``mimetype``
```

Comment:
We need a `versionadded` note and I would indent the bullets to match the content
```suggestion
    .. versionadded:: 5.2

    A named tuple to store attachments to an email.

    The named tuple has the following indexes:

    * ``filename``
    * ``content``
    * ``mimetype``
```